pkg/framework/plugins/nodeutilization/highnodeutilization.go

```diff
@@ -197,7 +188,7 @@ func (h *HighNodeUtilization) Balance(ctx context.Context, nodes []*v1.Node) *fr
 
 	lowNodes, schedulableNodes := nodeInfos[0], nodeInfos[1]
 
-	klog.V(1).InfoS("Criteria for a node below target utilization", h.criteria...)
+	klog.V(1).InfoS("Criteria for a node below target utilization", thresholdsToKeysAndValues(h.args.Thresholds)...)
```

Comment:
Originally, the output for InfoS was precomputed to avoid extra computation each time the plugin is run. Let's keep `criteria` field around and just `criteria=thresholdsToKeysAndValues(h.args.Thresholds)`.

Reply:
Do we call `Balance()` multiple times for the same plugin during the same deschedule run ?

Reply:
My point here is that between executions we need to read/parse/reload the descheduler policy file and when we do that we may potentially have different thresholds.  This performance gain (populate `criteria` only when the plugin is instantiated) can be accomplished only if the whole pod goes down when the policy config change and while the latter does not change we don't create a new plugin instance. Please let me know where my assumptions are wrong.

Reply:
Done.